packages/modul-components/src/components/navbar/navbar.ts

```diff
@@ -264,15 +271,15 @@ export class MNavbar extends BaseNavbar implements Navbar {
     }
 
     private get buttonSkin(): string {
-        return this.skin === MNavbarSkin.NavMain || this.skin === MNavbarSkin.NavSub || this.skin === MNavbarSkin.NavSoft || this.skin === MNavbarSkin.TabDark ? 'dark' : 'light';
+        return this.skin === MNavbarSkin.NavMain || this.skin === MNavbarSkin.NavSub || this.skin === MNavbarSkin.NavSoft || this.skin === MNavbarSkin.TabDark || this.skin === MNavbarSkin.TabDarkMain ? 'dark' : 'light';
```

Comment:
Pourrait être écrit sous cette forme :
[MNavbarSkin.NavMain, MNavbarSkin.NavSub, MNavbarSkin.NavSoft, MNavbarSkin.TabDark, MNavbarSkin.TabDarkMain].includes(this.skin) ? 'dark' : 'light'